### ISSUE TITLE
Cargo install does not work

### ISSUE THREAD

**Issue by hendrikmaus** [hendrikmaus/helm-templexer#24]
Why does `cargo install helm-templexer` fail?

```log
    Updating crates.io index
error: could not find `helm-templexer` in registry `https://github.com/rust-lang/crates.io-index` with version `*`
```

**Comment by IKoshelev** [comment]
Having the same issue 
```
could not find `Rng` in registry `https://github.com/rust-lang/crates.io-index` with version `*`
```

**Comment by hendrikmaus** [comment]
Thanks for the hint; how did you get the log entry?
I can neither find `Rng` in the `Cargo.lock` file, nor on crates.io - will have to find out where it comes from.

**Comment by IKoshelev** [comment]
Revisiting the issue, it looks like I may have needed `rng` instead of `Rng`.

**Comment by hendrikmaus** [comment]
Working on this now.

I tried to update all dependencies first; published `v0.2.1-alpha`, but a bare `cargo install helm-templexer` still fails.

However, `cargo install helm-templexer --version 0.2.1-alpha` *works*.

I am wondering if the issue is due to the `alpha` in the version. I reckon I will try to remove it. Having the version still start with a `0` for major should be enough to indicate that age/maturity of the tool.

**Comment by hendrikmaus** [comment]
And indeed the `alpha` in the version was the problem. I published `v0.2.1` and now `cargo install helm-templexer` works as expected.
